CHANGES.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## 6.3 (2024-04-25)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/6.2...6.3)
+
+**FEATURE**
+ - **sct_deepseg**: Add (internal, semi-private) support for `sct_deepseg` GPU inference on Linux. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4421)
```

Comment:
We probably don't want to advertise this as the top feature of this release, aha. :sweat_smile: